### ISSUE TITLE
Windows 10 : install

### ISSUE THREAD

**Issue by Hamahmi** [parlance/ctcdecode#117]
`pip install .` fails on Windows 10 with VS2019, the output goes as follows :
``
ERROR: Command "'D:\Conda\envs\win\python.exe' -u -c 'import setuptools, tokenize;__file__='"'"'C:\\Users\\Hamahmi~1\\AppData\\Local\\Temp\\pip-req-build-nm0q0bby\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\Hamahmi~1\AppData\Local\Temp\pip-record-n2_dv1h1\install-record.txt' --single-version-externally-managed --compile" failed with error code 1 in C:\Users\Hamahmi~1\AppData\Local\Temp\pip-req-build-nm0q0bby\
``

**Comment by CracKCatZ** [comment]
@Hamahmi did you fixed you error?

**Comment by Hamahmi** [comment]
> @Hamahmi did you fixed your error?

@CracKCatZ No, still not fixed, I switched to a VM running linux and also tried WSL2 for GPU support

**Comment by CracKCatZ** [comment]
Hey @Hamahmi and did wsl2 fixed your problem?

**Comment by Hamahmi** [comment]
@CracKCatZ  yes it did, not as efficient as working with a pure Linux installation but it does the job

**Comment by CracKCatZ** [comment]
@Hamahmi ok thanks for the fast response:)

**Comment by CracKCatZ** [comment]
@Hamahmi could you maybe tell me your solution of how to fix this problem? If you want you can add me on discord:SheeeshForce#8083

**Comment by Hamahmi** [comment]
@CracKCatZ as I mentioned, just install WSL (now 2 actually) on your windows and start from there as if it was a linux set up instead of windows

**Comment by CracKCatZ** [comment]
@Hamahmi ok thank you I already fixed it by installing kali Linux. I have one question. Is the kenlm required to test the speechrecognition??? Thanks for your answer and help in advance:)
